src/ocean/util/container/queue/NotifyingQueue.d

```diff
@@ -617,9 +619,9 @@ class NotifyingQueue ( T ) : NotifyingByteQueue
                 return null;
             }
 
-            auto void_buffer = cast(void[]) data;
+            Const!(void[]) void_buffer = data;
 
-            Deserializer.deserialize!(T)(void_buffer, cont_buffer);
+            Deserializer.deserialize(void_buffer, cont_buffer);
```

Comment:
It's a bit silly: in `Deserializer` there are
```d
static Contiguous!(S) deserialize ( S ) ( ref void[] src )
static Contiguous!(S) deserialize ( S ) ( in void[] src, ref Contiguous!(S) dst )
```
Commenting out the first, which doesn't work with `ubyte[] src`, fixes this compile error and works with `ubyte[] src`.